### PR TITLE
v1.0.1 release

### PR DESCRIPTION
**Pull request by StokesMIDE**

### CODE REVIEW

**Comment by CFlaniganMide** [comment]
I think the base branch should be main instead of development.

**Review by SamRagusa — DISMISSED** [review]

File: endaq/ide/info.py [hunk 1]
```diff
@@ -319,10 +319,10 @@ def to_pandas(
...
+    
+    if hasattr(channel, "subchannels"):
+        columns = [sch.name for sch in channel.subchannels]
+    else:
+        columns = [channel.name]
 
-    result = pd.DataFrame(
-        data, index=t, columns=[sch.name for sch in channel.subchannels]
-    )
-    result.index.name = "timestamp"
```
Comment (SamRagusa) [inline endaq/ide/info.py:326]: Should this be pluralized?   I'm not saying it should, just asking

File: .github/workflows/unit-tests.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+jobs:
+  unit-tests:
+    name: Run Unit Tests
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
```
Comment (SamRagusa) [inline .github/workflows/unit-tests.yml:14]: Should non-Windows OS be listed here?

File: endaq/ide/info.py (hunk 1 above)
Comment (StokesMIDE) [reply endaq/ide/info.py:326]: Which, `channel.subchannels`? That's correct.

**Comment by SamRagusa** [comment]
I posed 2 questions, but other than those everything looks good. 

The version number increments the micro revision rather than the minor revision, but after a discussion with David that seems purposeful (in which case I'll create the v1.1.0 release when the endaq.plot officially goes to 1.1.0)

**Review by SamRagusa — APPROVED** [review]
I'm approving for the situation that my questions had answers of 'no, this is how it should be'.

File: endaq/ide/info.py (hunk 1 above)
Comment (SamRagusa) [reply endaq/ide/info.py:326]: I was referring to the column title "timestamp"

Comment (SamRagusa) [reply endaq/ide/info.py:326]: Actually as I say out loud the other titles, they would be not pluralized.    So I think the answer to my original question would be 'no' haha

File: .github/workflows/unit-tests.yml (hunk 2 above)
Comment (CrepeGoat) [reply .github/workflows/unit-tests.yml:14]: yeah we're using this code in Colab's and potentially cloud work in the future, so the unit tests ensuring functionality should be run on, in addition to Windows, at least also Linux. Adding macOS isn't directly necessitated, but imo we may as well just to be well-rounded. (Also I often develop on macOS, so I personally would appreciate it.)

**Comment by StokesMIDE** [comment]
Apologies for the false start. Another PR has been created (#46) with the correct branch names and versions.

**Pull request closed without merging**